doc/sphinx/language/core/records.rst

```diff
@@ -281,22 +256,52 @@ There are currently two ways to introduce primitive records types:
 #. Through the ``Inductive`` and ``CoInductive`` commands, when
    the :term:`body` of the definition is a record declaration of the form
    ``Build_``\ `R` ``{`` |p_1| ``:`` |t_1|\ ``; … ;`` |p_n| ``:`` |t_n| ``}``.
-   In this case the types can be recursive and eta-conversion is disallowed. These kind of record types
-   differ from their traditional versions in the sense that dependent
+   In this case the types can be recursive and eta-conversion is disallowed.
+   These kind of record types differ from their traditional versions in the
+   sense that dependent
    elimination is not available for them and only non-dependent case analysis
    can be defined.
 
+For both cases the :flag:`Primitive Projections` :term:`flag` must be set or
+the :attr:`projections(primitive)` :term:`attribute`  must be supplied.
+
+.. flag:: Primitive Projections
+
+   This :term:`flag` turns on the use of primitive projections when defining
+   subsequent records (even through the ``Inductive`` and ``CoInductive``
```

Comment:
```suggestion
   subsequent records (even through the :cmd:`Inductive` and :cmd:`CoInductive`
```

Reply:
There were quite a few of these littered around the document, I've changed as many as I could find.